index.html

```diff
@@ -1,22 +1,38 @@
-<!DOCTYPE html>
+<!doctype html>
 <html lang="fr">
-<head>
-  <meta charset="utf-8">
-  <title>Ecobalyse</title>
-  <link rel="canonical" href="https://ecobalyse.beta.gouv.fr/">
-  <meta name="viewport" content="width=device-width, initial-scale=1">
-  <meta name="description" content="Accélerer la mise en place de l'affichage environnemental">
-  <meta name="theme-color" content="#333333">
-  <link rel="icon" type="image/svg" href="./public/img/logo.svg">
-  <link rel="preload" href="./public/fonts/Marianne-Regular.woff2" as="font" type="font/woff2" crossorigin>
-  <link rel="preload" href="./public/fonts/Marianne-Bold.woff2" as="font" type="font/woff2" crossorigin>
-  <link rel="stylesheet" href="./public/icomoon/style.css">
-  <link rel="stylesheet" href="./styles.scss">
-  <script type="module" src="index.js"></script>
-</head>
-<body>
-  <noscript>
-    <img src="https://stats.data.gouv.fr/matomo.php?idsite=196&amp;rec=1" style="border:0" alt="" />
-  </noscript>
-</body>
+  <head>
+    <meta charset="utf-8" />
+    <title>Ecobalyse</title>
+    <link rel="canonical" href="https://ecobalyse.beta.gouv.fr/" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <meta name="description" content="Accélerer la mise en place de l'affichage environnemental" />
+    <meta name="theme-color" content="#333333" />
```

Comment:
Could you please exclude html formatting from the configuration? In my experience, sometimes prettier messes with implicit white spaces when doing so. Thanks!

Reply:
✅